plugin/src/query/parser.ts

```diff
@@ -4,6 +4,7 @@ import YAML from "yaml";
 import { z } from "zod";
 
 type ErrorTree = string | { msg: string; children: ErrorTree[] };
+const MIN_COMPLETED_TASKS_AUTOREFRESH = 9;
```

Comment:
This looks like a call to the completed tasks API endpoint counts as a 'full sync' operation:

> For each user, you can make a maximum of 1000 partial sync requests within a 15 minute period.
> 
> For each user, you can make a maximum of 100 full sync requests within a 15 minute period.

Is this being conservative or have you verified which kind this request count as?

Its also worth noting the utility of this is somewhat lessened since a user can have any number of blocks rendered at once